### PR TITLE
Adaptive breakpoints 0.0.3 release

### PR DESCRIPTION
**Pull request by JoseAlba**
Removed Adaptive Container widget since it will be hosted in a different package.

### CODE REVIEW

File: adaptive_breakpoints/test/adaptive_breakpoints_test.dart [hunk 1]
```diff
@@ -39,249 +39,436 @@ void main() {
...
+///
+/// The AdaptiveContainer does everything a normal container does but with
+/// adaptive constraints. For more information go to one of the links below.
+///
+/// https://api.flutter.dev/flutter/widgets/Container-class.html
+/// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+///
+/// This class is useful whenever you want a container to only be active in
+/// certain [AdaptiveWindowType].
+class _AdaptiveContainer extends StatelessWidget {
```
Comment (rami-a) [inline adaptive_breakpoints/test/adaptive_breakpoints_test.dart]: Instead of recreating the _AdaptiveContainer in the tests, could you modify/simplify the tests to purely test the adaptive breakpoints functionality instead?

That way the tests that rely on `AdaptiveContainer` can instead move to the new package you create?

**Review by rami-a — APPROVED** [review]

File: adaptive_breakpoints/test/adaptive_breakpoints_test.dart (hunk 1 above)
Comment (JoseAlba) [reply adaptive_breakpoints/test/adaptive_breakpoints_test.dart]: Good idea

File: adaptive_breakpoints/lib/src/adaptive_breakpoints.dart [hunk 2]
```diff
@@ -15,18 +15,18 @@ import 'package:flutter/material.dart';
 /// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
 enum AdaptiveWindowType {
   xs,
-  s,
-  m,
-  l,
+  sm,
```
Comment (johnsonmh) [inline adaptive_breakpoints/lib/src/adaptive_breakpoints.dart]: Probably shouldn't be part of this change, but you should consider using the full words that are in the material guidance. As in `small`, `medium`.

This would fit better with the dart style guide of avoiding abbreviations unless the abbreviations are more common than the word itself: https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations

**Review by johnsonmh — APPROVED** [review]
LGTM

File: adaptive_breakpoints/lib/src/adaptive_breakpoints.dart (hunk 2 above)
Comment (JoseAlba) [reply adaptive_breakpoints/lib/src/adaptive_breakpoints.dart]: Done
